adminapi/cmdline/query.py

```diff
@@ -25,9 +25,11 @@ def main():
 
     attrs = options.attrs if options.attrs else ('hostname', )
```

Comment:
`('hostname',)` would not be a tuple anymore? I'm talking about just the space, not the comma here. ;)